tests/Lewee.Application.Tests.Unit/Lewee.Application.Tests.Unit.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.TestHost" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" />
+    <PackageReference Include="Microsoft.Extensions.Diagnostics.Testing" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Lewee.Application\Lewee.Application.csproj" />
+    <ProjectReference Include="..\..\src\Lewee.Domain\Lewee.Domain.csproj" />
```

Comment:
I don't think this reference is needed as `Lewee.Application` should refeence `Lewee.Domain`